bindings/php/opendal-php.stubs.php

```diff
@@ -0,0 +1,7 @@
+<?php
```

Comment:
It's required that all files must container Apache license headers to.

You can use `docker run -it --rm -v $(pwd):/github/workspace ghcr.io/korandoru/hawkeye-native fix` to fix them

Reply:
thank you @Xuanwo, updated it now.